### ISSUE TITLE
Create user not working

### ISSUE THREAD

**Issue by dborkan** [igarape/copcast-admin#283]
Seen on dev.mycopcast.com - when you click create user, the screen seems to freeze and I see this error:
```
Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!
```
followed by a lot lot more error text

**Comment by brunosiqueira** [comment]
@dborkan we deployed on Friday a corrupted version and corrected it a few minutes later. I am guessing that was the one you found this error. Can you check it again?

**Comment by arthurvasconcelos** [comment]
Yes this error was about this bug that was introduced with the translation commit 9efaa2c3ecfd929cb7faf50f82a3a92a2b5dd73d. I will close this issue, since this bug was fixed in commit 8e10bf0459cc08a2263dc1624ca5f42c8baa6663.

**Comment by dborkan** [comment]
Yup looks fixed now, thanks!

On Mon, Jun 6, 2016 at 9:46 AM, Arthur Vasconcelos <notifications@github.com
